stanford-sw/sw_index.properties

```diff
@@ -1,195 +1,198 @@
+era_facet = custom, removeTrailingPunct(650y:651y, [\\\\,;], ([A-Za-z0-9]{2}) )
+
+# Publication Fields
+pub_search = custom, getPublication
+vern_pub_search = custom, getLinkedField(260ab:264ab)
+pub_country = 008[15-17]:008[15-16], country_map.properties, first
+# publication dates
+# deprecated
+pub_date = custom, getPubDate
+pub_date_sort = custom, getPubDateSort
+pub_year_tisim = custom, getPubDateSliderVals
+# from 008 date 1
+publication_year_isi = custom, get008Date1(est)
+beginning_year_isi = custom, get008Date1(cdmu)
+earliest_year_isi = custom, get008Date1(ik)
+earliest_poss_year_isi = custom, get008Date1(q)
+release_year_isi = custom, get008Date1(p)
+reprint_year_isi = custom, get008Date1(r)
+other_year_isi = custom, getOtherYear
+# from 008 date 2
+ending_year_isi = custom, get008Date2(dm)
+latest_year_isi = custom, get008Date2(ik)
+latest_poss_year_isi = custom, get008Date2(q)
+production_year_isi = custom, get008Date2(p)
+original_year_isi = custom, get008Date2(r)
+copyright_year_isi = custom, get008Date2(t)
+# from 260c
+imprint_display = custom, getImprint
+
+# Date field for new items feed
+date_cataloged = custom, getDateCataloged
+
+language = custom, getLanguages, language_map.properties
+
+# old format field, left for continuity in UI URLs for old formats
+format = custom, getOldFormats
+format_main_ssim = custom, getMainFormats
+format_physical_ssim = custom, getPhysicalFormats
+genre_ssim = custom, getAllGenres
+
+db_az_subject = custom, getDbAZSubjects, db_subjects_map.properties
+
+physical = 300abcefg
+vern_physical = custom, getLinkedField(300abcefg)
+
+toc_search = 905art:505art
+vern_toc_search = custom, getLinkedField(505art)
+context_search = 518a
+vern_context_search = custom, getLinkedField(518a)
+summary_search = 920ab:520ab
+vern_summary_search = custom, getLinkedField(520ab)
+award_search = 986a:586a
+
+# URL Fields
+url_fulltext = custom, getFullTextUrls
+url_suppl = custom, getSupplUrls
+url_sfx = 956u, (pattern_map.sfx)
+url_restricted = custom, getRestrictedUrls
+
+# Standard Number Fields
+isbn_search = custom, getUserISBNs
+# Added fields for searching based upon list from Kay Teel in JIRA ticket INDEX-142
+issn_search = 022a:022l:022m:022y:022z:400x:410x:411x:440x:490x:510x:700x:710x:711x:730x:760x:762x:765x:767x:770x:771x:772x:773x:774x:775x:776x:777x:778x:779x:780x:781x:782x:783x:784x:785x:786x:787x:788x:789x:800x:810x:811x:830x, (pattern_map.issn)
+isbn_display = custom, getISBNs
+issn_display = custom, getISSNs
+lccn = 010a:010z, (pattern_map.lccn), first
+oclc = custom, getOCLCNums
+
+# Call Number Fields
+callnum_facet_hsim = custom, getCallNumHierarchVals(|, callnumber_map)
+callnum_search = custom, getLocalCallNums
+shelfkey = custom, getShelfkeys
+reverse_shelfkey = custom, getReverseShelfkeys
+
+# Location facet
+location_facet = custom, getLocationFacet
+
+# Stanford theses and dissertations facet
+stanford_theses_facet_hsim = custom, getStanfordThesesFacet
```

Comment:
totally not worth redoing, but this change should have gone with the commit for the relevant code changes.  This commit is now really doing 2 changes, but the description only mentions one.